### ISSUE TITLE
UI: repair button inactive by default

### ISSUE THREAD

**Issue by a1exsh** [thelastpickle/cassandra-reaper#365]
Using v1.1.0, in the UI the [Repair] button stays inactive even after you fill mandatory fields for Keyspace and Owner.  Only after clicking on the cluster drop-down it becomes active.  But normally you don't do that if you only have 1 cluster configured, or the drop-down was already showing the one you want to repair now.

**Comment by kotaroyamazaki** [comment]
This ticket seems to be already done.

**Comment by a1exsh** [comment]
Confirmed: no longer an issue with the version we currently use (1.2.1).
